src/utils/daos_hdlr.c

```diff
@@ -1673,9 +1757,11 @@ fs_copy_dir(struct cmd_args_s *ap,
 
 	/* create the destination directory if it does not exist */
 	rc = file_mkdir(ap, dst_file_dfs, dst_path, &tmp_mode_dir);
-	if (rc != EEXIST && rc != 0)
+	if (rc == EEXIST) {
+		DH_PERROR_SYS(ap, rc, "Directory '%s' exists", dst_path);
```

Comment:
@daltonbohning - what's the right thing here?  Log an error and continue or something else?  It doesn't look like either the NLT or Fi testing cover this, although we could easily make that change.

Reply:
@ashleypittman I like to use `cp`  as a reference, since that's what people are familiar with:
```
$ mkdir -p aaa/bbb
$ mkdir -p ddd/aaa/bbb
$ cp -r aaa ddd
# no output
$ diff -r aaa ddd/aaa
# no output, equal
```
Although `mpifileutils/dcp` does log an info message. (It also always copies INTO an existing directory.)
```
$ mkdir -p aaa/bbb
$ mkdir -p ddd/aaa/bbb
$ dcp aaa ddd
Original directory exists, skip the creation: `/tmp/dbohninx/test/ddd/aaa'
Original directory exists, skip the creation: `/tmp/dbohninx/test/ddd/aaa/bbb'
```
I think either approach here is okay, since if we fail to create/write files in those directories, we'll get a real error.

Reply:
This actually is a warning to me in stead of an error.

Reply:
Why is the output above not including the DH_PERROR_SYS() call from the PR?